frontend/src/components/ProductView/ProductButtons.tsx

```
@@ -0,0 +1,69 @@
+            {t('product.addToBag')}
+          </span>
+        </div>
+      </button>
+
+      {/* Comprar por WhatsApp */}
+      <button
+        onClick={handleBuyWhatsApp}
+        className="flex flex-row justify-center items-center w-full h-12 min-h-[48px]"
+      >
+        <div className="flex flex-row justify-center items-center px-4 py-2.5 gap-2 bg-[#215E6B] rounded-full w-full transition-all hover:opacity-80">
+          <svg className="w-5 h-5 flex-shrink-0" viewBox="0 0 24 24" fill="white">
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)